pkg/asset/installconfig/ovirt/datacenter.go

```diff
@@ -0,0 +1,25 @@
+package ovirt
+
+import (
+	ovirtsdk4 "github.com/ovirt/go-ovirt"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// datacentersAvailable look for all datacenters available in the system.
+// Users can provide the criteria for the search. Example: status=up
```

Comment:
Can't we have status=up as the default, and let the user override this?

Reply:
Sure, let's do it.